internal/common/commontest/request_gen.go

```diff
@@ -0,0 +1,156 @@
+package commontest
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+)
+
+// UnsafeRequests generates a list of requests for all unsafe
+// endpoints common to all app frameworks.
+func UnsafeRequests(addr string) ([]*http.Request, error) {
+	var reqs []*http.Request
+	type genFunc func(addr string) ([]*http.Request, error)
```

Comment:
naming nit: I think `Func` in `genFunc` stutters a bit with the type which follows it. It kinda reads to me like `type gen func func`, and its name could just be gen (especially since it's only used in this scope). The loop variable `gen` could be `input` or `makeInput` or `f` or something if you're worried about shadowing.

Reply:
renamed `genFunc` to `generator`